Misc/NEWS.d/next/Core and Builtins/2018-09-20-15-41-58.bpo-34751.Yiv0pV.rst

```diff
@@ -0,0 +1,3 @@
+The hash function for tuples is now based on xxHash.
+This makes hash collisions less likely.
```

Comment:
Suggest instead "This makes a high rate of collisions much less likely in certain rare cases on all platforms, and on 64-bit platforms improves tuple hashes in general (the old algorithm was designed for 32-bit hash codes)."

Reply:
I don't see why you think that it's a 32-bit vs. 64-bit issue. The statement "the old algorithm was designed for 32-bit hash codes" is probably true. However, that's not what is causing the problems here.

Maybe I could mention that we now have different algorithms for 32-bit and 64-bit systems, that might be relevant.

Reply:
As you pointed out on BPO, the multiplier was "too small" for 64-bit boxes before.  For that reason, the new algorithm (which doesn't have that issue) can be expected to do better on 64-bit boxes _in general_, quite apart from the handful of very bad special all-platform cases we identified.  That's potentially of real interest to users.  That the new algorithm is merely somewhat different between its own 32- and 64-bit versions is an implementation detail of no interest to the vast bulk of users.